### ISSUE TITLE
[Lab3] Προβλήματα με το accuracy και το loss

### ISSUE THREAD

**Issue by theopnt** [slp-ntua/slp-labs#69]
Καλησπέρα,

Στην προπαρασκευή  δεν επιτυγχάνουμε accuracy μεγαλύτερο από 0.7 μετά από πολλές εποχές. Χρησιμοποιούμε CrossEntropyLoss ως loss function και Adam ως optimizer. Το τελικό accuracy επίσης δεν αλλάζει με όλες τις τιμές learning rate και batch size που έχουμε δοκιμάσει. Τι accuracy θεωρείται αποδεκτό? 

~~Στο LSTM μας επίσης δεν μεταβάλλεται καθόλου το accuracy και το loss για όλα τα epochs που τρέχουμε. Χρησιμοποιούμε τα ίδια με την προπαρασκευή ως loss function και optimizer. Τι θα μπορούσαμε να κάνουμε για να διορθώσουμε το πρόβλημα αυτό?~~

Το δεύτερο πρόβλημα λύθηκε

**Comment by pxaris** [comment]
Καλησπέρα,

Περίπου τόση αναμένεται να είναι η επίδοση του DNN στο "Movie Review" data set. Θα δείτε, επίσης, το train loss να μειώνεται ενώ το test loss παραμένει στα ίδια περίπου επίπεδα. Η επίτευξη της καλύτερης δυνατής επίδοσης είναι έξω από τα πλαίσια της παρούσας εργαστηριακής άσκησης. Οπότε, θα πρότεινα να δώσετε τον χρόνο σας στα επόμενα ερωτήματα από το να προσπαθήσετε να "κουρδίσετε" τις υπερπαραμέτρους.

**Comment by theopnt** [comment]
Σας ευχαριστώ πολύ
